README.md

```diff
@@ -23,6 +23,7 @@ Configuration options for fluent.conf are:
 * `log_key` - Used to specify the key when merging json or sending logs in text format (default `message`)
 * `open_timeout` - Set timeout seconds to wait until connection is opened.
 * `add_timestamp` - Add `timestamp` field to logs before sending to sumologic (default `true`)
+* `proxy_uri` - Add the `uri` of the `proxy` environment if present.@michnmi looks good. Can you update the readme and also allow the ENV setting and then we can merge and cut a new release.
```

Comment:
Looks like the comments got appended to the README.  If you remove those I can merge and get a new release out.